src/Compilers/CSharp/Portable/Lowering/LambdaRewriter/SynthesizedClosureMethod.cs

```diff
@@ -116,6 +116,11 @@ private static DeclarationModifiers MakeDeclarationModifiers(ClosureKind closure
                 mods |= DeclarationModifiers.Async;
             }
 
+            if (originalMethod.IsExtern)
+            {
+                mods |= DeclarationModifiers.Extern;
```

Comment:
>DeclarationModifiers.Extern; [](start = 24, length = 28)

Should we override IsExtern for SynthesizedMethodBaseSymbol instead?

Reply:
If we did, we should do the same for similar properties like IsAsync. It doesn't seem very desirable to make such a change because we already receive the 'flags' field and sealed overrides from SourceMemberMethodSymbol.

---
In reply to: [354058007](https://github.com/dotnet/roslyn/pull/40011#discussion_r354058007) [](ancestors = 354058007)